src/PhoneNumber/PhoneNumber.js

```diff
@@ -179,13 +107,16 @@ class PhoneNumber extends Component {
       onBlur,
       onChange,
       onFocus,
+      prefix,
       readOnly,
       required,
       value,
       ...otherProps
     } = this.props;
 
-    const { formattedNumber, isFocused, selectedCountry } = this.state;
+    const { dialingCode, phoneNumber, isFocused } = this.state;
+
+    const countryPrefix = dialingCode === '' ? prefix : dialingCode;
```

Comment:
I don't understand the difference between prefix and dialingCode at this point, on top of this file we do (line 46):
`this.state = {
      dialingCode: prefix
}`
So i'm really confused about this line here.

Reply:
Also i don't see any test that assert that this behavior works as expected

Reply:
The prefix is the default value if it exists. The dialing code is the reference in the internal state

Reply:
Fixed